model/articles.model.js

```diff
@@ -1,5 +1,25 @@
 const db = require("../db/connection");
-const fs = require("fs/promises");
+
+function fetchAllArticles(request, response) {
```

Comment:
Whilst this approach will work, there are some problems with this approach and it is not the most efficient way of getting a comment_count for your articles. 

1) Permanently altering the db is something we would generally try to avoid, so do try to avoid ALTER TABLE when you can achieve the same results with an aggregate function. 

2) Also consider that this model needs to query the db 3 times in order to get the required information. This makes it slower and less efficient (and in some cases costly!) 

Try to think of an alternative approach - you've already shown that you know how to apply an aggregate function like COUNT - you just need to think about what it should GROUP BY 

I'd recommend you refactor this endpoint to take these considerations into account before you host this project (more info on hosting later this week!)

Great job though - it certainly works!